### PR TITLE
chore: add local-integ-test make target for faster integration tests

### PR DESCRIPTION
**Pull request by bvtujo**
Some of our integ tests take 30 minutes to run. This adds a make target which excludes those long-running tests. 

Accepting suggestions for names of this make target. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by Lou1415926 — COMMENTED** [review]
lgtm I'd say :shipit: 👀

File: internal/pkg/deploy/cloudformation/cc_pipeline_integration_test.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// +build integration
+// +build integration,deploy
```
Comment (efekarakus) [inline internal/pkg/deploy/cloudformation/cc_pipeline_integration_test.go]: nit: I like the idea! can we instead change the tag to `integration-remote` to be more specific? I feel like it's possible for us to use the `deploy` build tag for possibly another scenario

Comment (bvtujo) [reply internal/pkg/deploy/cloudformation/cc_pipeline_integration_test.go]: Good idea! Done.

**Comment by bvtujo** [comment]
results of `make local-integ-test` with the new changes:
```
Packaging static files
go test -race -count=1 -timeout=60m -tags=integration,!integration-remote ./internal...
ok      github.com/aws/copilot-cli/internal/pkg/addon   0.168s
?       github.com/aws/copilot-cli/internal/pkg/addon/mocks     [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/aws/aas 0.105s
?       github.com/aws/copilot-cli/internal/pkg/aws/aas/mocks   [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/aws/cloudformation      0.133s
?       github.com/aws/copilot-cli/internal/pkg/aws/cloudformation/mocks        [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/aws/cloudformation/stackset     0.095s
?       github.com/aws/copilot-cli/internal/pkg/aws/cloudformation/stackset/mocks       [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/aws/cloudwatch  0.052s
?       github.com/aws/copilot-cli/internal/pkg/aws/cloudwatch/mocks    [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/aws/cloudwatchlogs      0.058s
?       github.com/aws/copilot-cli/internal/pkg/aws/cloudwatchlogs/mocks        [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/aws/codepipeline        0.067s
?       github.com/aws/copilot-cli/internal/pkg/aws/codepipeline/mocks  [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/aws/codestar    0.061s
?       github.com/aws/copilot-cli/internal/pkg/aws/codestar/mocks      [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/aws/ec2 0.085s
?       github.com/aws/copilot-cli/internal/pkg/aws/ec2/mocks   [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/aws/ecr 0.077s
?       github.com/aws/copilot-cli/internal/pkg/aws/ecr/mocks   [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/aws/ecs 0.117s
?       github.com/aws/copilot-cli/internal/pkg/aws/ecs/mocks   [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/aws/iam 0.071s
?       github.com/aws/copilot-cli/internal/pkg/aws/iam/mocks   [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/aws/identity    0.078s
?       github.com/aws/copilot-cli/internal/pkg/aws/identity/mocks      [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/aws/profile     0.050s
ok      github.com/aws/copilot-cli/internal/pkg/aws/resourcegroups      0.085s
?       github.com/aws/copilot-cli/internal/pkg/aws/resourcegroups/mocks        [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/aws/route53     0.041s
?       github.com/aws/copilot-cli/internal/pkg/aws/route53/mocks       [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/aws/s3  0.101s
?       github.com/aws/copilot-cli/internal/pkg/aws/s3/mocks    [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/aws/secretsmanager      0.048s
?       github.com/aws/copilot-cli/internal/pkg/aws/secretsmanager/mocks        [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/aws/sessions    0.079s
ok      github.com/aws/copilot-cli/internal/pkg/aws/tags        0.069s
ok      github.com/aws/copilot-cli/internal/pkg/cli     1.923s
?       github.com/aws/copilot-cli/internal/pkg/cli/group       [no test files]
?       github.com/aws/copilot-cli/internal/pkg/cli/mocks       [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/config  13.960s
?       github.com/aws/copilot-cli/internal/pkg/config/mocks    [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/deploy  0.285s
ok      github.com/aws/copilot-cli/internal/pkg/deploy/cloudformation   0.410s
?       github.com/aws/copilot-cli/internal/pkg/deploy/cloudformation/mocks     [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/deploy/cloudformation/stack     2.318s
?       github.com/aws/copilot-cli/internal/pkg/deploy/cloudformation/stack/mocks       [no test files]
?       github.com/aws/copilot-cli/internal/pkg/deploy/mocks    [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/describe        0.153s
?       github.com/aws/copilot-cli/internal/pkg/describe/mocks  [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/ecs     0.066s
?       github.com/aws/copilot-cli/internal/pkg/ecs/mocks       [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/exec    1.873s
?       github.com/aws/copilot-cli/internal/pkg/exec/mocks      [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/ini     0.037s
ok      github.com/aws/copilot-cli/internal/pkg/initialize      0.073s
?       github.com/aws/copilot-cli/internal/pkg/initialize/mocks        [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/list    0.070s
?       github.com/aws/copilot-cli/internal/pkg/list/mocks      [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/logging 1.080s
?       github.com/aws/copilot-cli/internal/pkg/logging/mocks   [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/manifest        0.185s
ok      github.com/aws/copilot-cli/internal/pkg/repository      0.032s
?       github.com/aws/copilot-cli/internal/pkg/repository/mocks        [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/stream  0.349s
ok      github.com/aws/copilot-cli/internal/pkg/task    0.066s
?       github.com/aws/copilot-cli/internal/pkg/task/mocks      [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/template        3.858s
?       github.com/aws/copilot-cli/internal/pkg/template/mocks  [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/term/color      0.026s
?       github.com/aws/copilot-cli/internal/pkg/term/command    [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/term/cursor     0.023s
ok      github.com/aws/copilot-cli/internal/pkg/term/log        0.047s
ok      github.com/aws/copilot-cli/internal/pkg/term/progress   0.394s
?       github.com/aws/copilot-cli/internal/pkg/term/progress/mocks     [no test files]
ok      github.com/aws/copilot-cli/internal/pkg/term/prompt     0.046s
ok      github.com/aws/copilot-cli/internal/pkg/term/selector   0.112s
?       github.com/aws/copilot-cli/internal/pkg/term/selector/mocks     [no test files]
?       github.com/aws/copilot-cli/internal/pkg/version [no test files]
```

File: Makefile [hunk 2]
```diff
@@ -114,7 +114,13 @@ run-integ-test:
 	# Also adding count=1 so the test results aren't cached.
 	# This command also targets files with the build integration tag
 	# and runs tests which end in Integration.
-	go test -race -count=1 -timeout 60m -tags=integration ${PACKAGES}
+	go test -race -count=1 -timeout 60m -tags=integration,integration-remote ${PACKAGES}
```
Comment (efekarakus) [inline Makefile]: do we  need to specify `integration-remote` here or can we just have `-tags=integration`

Comment (bvtujo) [reply Makefile]: hmm, maybe you're right--let me check

**Comment by bvtujo** [comment]
I think you can't use hyphens in tag names-- latest commit addresses that, and removes the unnecessary `!remote` tag in the `make integ-test` target.

**Review by efekarakus — APPROVED** [review]

**Review by iamhopaul123 — APPROVED** [review]

**Pull request merged**